### DISCUSSION TITLE
How to know which model/settings to use

### DISCUSSION THREAD

**Discussion by ySergi** [Anjok07/ultimatevocalremovergui#647] [Q&A]
Hello, is there any documentation for each model and settings? There are many and I do not know what they are for or which ones to use, I am new using this program. I see that there is no Wiki of this program, it would be nice if there was so it would be much more accessible to inexperienced people like me.

**Comment by drNn22farahat** [comment]
hello there, I have the same issue
if you find any resources plz share them with us.
